.github/workflow/pullrequest.yaml

```diff
@@ -0,0 +1,24 @@
+---
+name: CI
+on: [pull_request]
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    env:
+      HELM_VERSION: "v3.3.2"
```

Comment:
New version of Helm was recently released: `3.3.3`

Reply:
they _just_ released helm-3.3.4 :-)